### ISSUE TITLE
GPS flickering back and forth

### ISSUE THREAD

**Issue by aconite33** [xbenjii/PokeMock#7]
Android 6, Xposed framework, latest. Mock Mock module installed.

Runs perfectly fine, but GPS clicks back and forth between real GPS location and spoofed.

**Comment by xbenjii** [comment]
I have noticed this actually. It seems to happen if you're idling then the real GPS takes over.

**Comment by aconite33** [comment]
The only way I was able to stop it, was by getting an app that kills services (requires root) and stops the fuselocation service. But then that basically kills your entire GPS on your phone and requires a reboot.

**Comment by igromanru** [comment]
I've found the solution with disabling the FusedLocationService few days ago. After I have disabled it few times, my android started to crash until I managed to reactivate it. So be careful with it, it could break your phone.. We need a better solution.

**Comment by aconite33** [comment]
I actually had it working yesterday without disabling the fuse location service, but I'm not sure what I did. I was messing with a variety of spoofying gps apps (Fake GPS), putting it into /system/priv-app seemed to do the trick. But since xbenjii updated to use the joystick I messed something up, which led me to the fuse service solution.

**Comment by igromanru** [comment]
I've tested it several times with Fake GPS as you said (it's a commend solution in internet) and some other apps. But sooner or late Pokemon or also Google Maps get your true location, it's just a question of time.

**Comment by erdalgunes** [comment]
Please be sure you've selected "Device only" mode from location settings.

**Comment by aconite33** [comment]
Yep, even with Device Only, still will flicker back and forth between locations.

**Comment by igromanru** [comment]
Why people just won't understand, that switch to "Device only" don't solve it permanently. Got the same answer from the GPS Faker developer. 
I don't know how, but after a while google maps and Pokemon still get the true location.

**Comment by TheGianni** [comment]
I've had the same issue while using the FakeGPS app installed as system app. The strange thing was that it didn't happen all the time. I finally figured out that the flickering only appears if your device is in sight of the GPS satellites. For example, as long as I'm inside a building which has solid enough walls, or maybe down in the basement, it works perfectly fine without switching back to your real location. The moment I get outside on the balcony the flickering keeps coming back and you get softbanned. Also it only works if you enable the high-accuracy mode.
